.github/workflows/dco-check.yml

```diff
@@ -1,18 +0,0 @@
-name: Developer Certificate of Origin Check
```

Comment:
Why is DCO check removed?

Reply:
Because if you see the checks there is already a dco app doing the work.
The DCO workflow file is very old and a lot of repo already removed it.

Thanks.